.mailmap

```diff
@@ -7,6 +7,7 @@ franzTobiasDLR <tobias.franz@dlr.de>			Tobias Franz <tobias.franz@dlr.de>
 SaMuellerDLR <sa.mueller@dlr.de>				Müller, Sascha ( SC-SRV ) <sa.mueller@dlr.de>
 SaMuellerDLR <sa.mueller@dlr.de>				SaMuellerDLR <48356419+SaMuellerDLR@users.noreply.github.com>
 SaMuellerDLR <sa.mueller@dlr.de>				Sascha Müller <sa.mueller@dlr.de>
+SaMuellerDLR <sa.mueller@dlr.de>				muel_s8 <muel_s8@SC-010214.intra.dlr.de>
```

Comment:
Looks like your username slipped into the list of mail addresses. I don't know if this is an issue, but that probably isn't a legit mail address.